lib/core/presentation/pages/edit_profile/widgets/dropdown_widget.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:dropdown_button2/dropdown_button2.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class DropdownWidget extends StatelessWidget {
```

Comment:
Curious why not use existing widget or common widget we have ? and enhance that component instead of create new one

If design difference, can we just ask designer to align and update it @roncoderpush

Reply:
The list industry here will be pretty long and our `FrostedGlassDropdownWidget` are not supported changing offset  when the list is too long. This is a complicated case and I prefer using a well-known package to support that instead of enhance the class.
Notes: the background color of the list below will need to change later
![Simulator Screenshot - iPhone 14 Pro Max - 2023-09-13 at 11 14 35](https://github.com/lemonadesocial/lemonade-flutter/assets/108661349/bbff0d6c-1338-45c4-97e4-112606b97109)